server/controllers/notification.js

```diff
@@ -0,0 +1,108 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+
+const notifications = {
+  newRequest: {
+    title: "New Request",
+    message: "You recieved a new request from ",
+  },
+  message: {
+    title: "New Message",
+    message: "You received a new message from ",
+  },
+  requestResponse: {
+    title: "Response Received",
+    message: "You received a response from ",
+  },
+};
+
+// @route POST /notification/:id
+// @desc create notification
+// @access Private
+exports.createNotification = asyncHandler(async (req, res) => {
+  const sender = rep.params.sender;
+  const type = req.params.type;
+  const userId = req.params.userId;
+
+  if (!sender | type | userid) {
```

Comment:
Did you mean if any of these is missing, that's `if (!sender | !type | !userId)`? Also note the typo in `userId`

Reply:
Yes all typos. Thanks for finding, fixed.